datastream-server-api/src/main/java/com/linkedin/datastream/server/api/transport/DatastreamRecordMetadata.java

```diff
@@ -24,6 +25,21 @@ public DatastreamRecordMetadata(String checkpoint, String topic, int partition)
     _checkpoint = checkpoint;
     _topic = topic;
     _partition = partition;
+    _recordIndex = 0;
```

Comment:
nit: Should this be brooklinEventIndex instead? Since a Record is a collection of events?

Reply:
done